---a/net/minecraft/command/CommandShowSeed.java
+++b/net/minecraft/command/CommandShowSeed.java
@@ -5,34 +5,29 @@
 import net.minecraft.util.ChatComponentTranslation;
 import net.minecraft.world.World;
 
-public class CommandShowSeed extends CommandBase
-{
+public class CommandShowSeed extends CommandBase {
+
     private static final String __OBFID = "CL_00001053";
 
-    public boolean canCommandSenderUse(ICommandSender sender)
-    {
+    public boolean canCommandSenderUse(ICommandSender sender) {
         return MinecraftServer.getServer().isSinglePlayer() || super.canCommandSenderUse(sender);
     }
 
-    public String getName()
-    {
+    public String getName() {
         return "seed";
     }
 
-    public int getRequiredPermissionLevel()
-    {
+    public int getRequiredPermissionLevel() {
         return 2;
     }
 
-    public String getCommandUsage(ICommandSender sender)
-    {
+    public String getCommandUsage(ICommandSender sender) {
         return "commands.seed.usage";
     }
 
-    public void execute(ICommandSender sender, String[] args) throws CommandException
-    {
-        Object object = sender instanceof EntityPlayer ? ((EntityPlayer)sender).worldObj : MinecraftServer.getServer().worldServerForDimension(0);
-        sender.addChatMessage(new ChatComponentTranslation("commands.seed.success", new Object[] {Long.valueOf(((World)object).getSeed())}));
+    public void execute(ICommandSender sender, String[] args) throws CommandException {
+        Object object = sender instanceof EntityPlayer ? ((EntityPlayer) sender).worldObj : MinecraftServer.getServer().worldServerForDimension(0);
+        sender.addChatMessage(new ChatComponentTranslation("commands.seed.success", new Object[]{Long.valueOf(((World) object).getSeed())}));
     }
-}
+}
